frontend/ui/elements/filemanager_menu_order.lua

```diff
@@ -30,6 +30,15 @@ local order = {
         "time",
         -- end common settings
     },
+    network = {
+        "network_wifi",
+        "network_proxy",
+        "network_restore",
+        "network_info",
+        "network_before_wifi_action",
+        "----------------------------",
+        "SSH",
```

Comment:
Hm, uppercase doesn't really fit with the rest of the menu. Not really a hard rule, just something I'm noticing.

Reply:
yes me too, I hesitated, but SSH is really spelled with uppercase when I searched on google.

I can change it, no big deal.